pages/admin/permit-holder/[permitHolderId].tsx

```diff
@@ -1,95 +1,187 @@
+          ),
+        });
+      }
+    },
+    onError: error => {
+      toast({
+        status: 'error',
+        description: error.message,
+      });
+    },
+    refetchQueries: ['GetPermitHolder'],
+  });
+
+  // Submit edited user information mutation
+  const [submitEditedUserInformation] = useMutation<
+    UpdateApplicantResponse,
+    UpdateApplicantRequest
+  >(UPDATE_APPLICANT_MUTATION, {
+    onCompleted: data => {
+      if (data?.updateApplicant.ok) {
+        toast({
+          render: () => (
+            <SuccessfulEditAlert>{"User's information has been edited."}</SuccessfulEditAlert>
+          ),
+        });
+      }
+    },
+    onError: error => {
+      toast({
+        status: 'error',
+        description: error.message,
+      });
+    },
+  });
+
+  /**
+   * Update Doctor Information handler
+   * @param physicianData Updated physician data
+   */
+  const handleUpdateDoctorInformation = (physicianData: UpsertPhysicianInput) => {
+    submitEditedDoctorInformation({ variables: { input: { ...physicianData } } });
+  };
+
+  /**
+   * Update application handler
+   * @param applicationData Updated application data
+   */
+  const handleUpdateUserInformation = (applicantData: UpdateApplicantInput) => {
+    submitEditedUserInformation({ variables: { input: { ...applicantData } } });
+  };
 
   return (
     <Layout>
       <GridItem rowSpan={1} colSpan={12} marginTop={3}>
-        <PermitHolderHeader
-          applicant={applicant as unknown as Applicant}
-          applicationStatus={applicationStatus}
-        />
+        {applicantData && <PermitHolderHeader applicant={applicantData} />}
       </GridItem>
       <GridItem rowSpan={12} colSpan={5} marginTop={5} textAlign="left">
         <Stack spacing={5}>
-          <PersonalInformationCard applicant={applicant as unknown as Applicant} />
-          {/* Temporarily commented to pass CI checks for View Request page */}
-          {/* <DoctorInformationCard physician={physician} />
-          <GuardianInformationCard guardian={guardian} /> */}
+          {applicantData && (
+            <PersonalInformationCard
+              applicant={applicantData}
+              onSave={handleUpdateUserInformation}
+            />
+          )}
+          {data && previousPhysicianData && (
+            <DoctorInformationCard
+              physician={data.applicant.medicalInformation.physician}
+              previousPhysicianData={previousPhysicianData}
+              onSave={handleUpdateDoctorInformation}
+            />
+          )}
+          {data && <GuardianInformationCard guardian={data?.applicant.guardian} />}
         </Stack>
       </GridItem>
 
       <GridItem rowSpan={12} colSpan={7} marginTop={5} textAlign="left">
         <Stack spacing={5}>
-          <AppHistoryCard />
-          <AttachedFilesCard />
-          <MedicalHistoryCard />
+          {permits && <AppHistoryCard permits={permits} />}
+          {data && <AttachedFilesCard />}
```

Comment:
```suggestion
          <AttachedFilesCard />
```
Remove this for now. We will add the conditional rendering logic later